packages/core/src/utils/setup-package-json.test.ts

```diff
@@ -158,23 +158,6 @@ describe('diffPackageJson', () => {
       expectSnapshots({ diffs, packageJson, expectedPackageJson });
     });
 
-    test('exports out of order', () => {
```

Comment:
If the CSS last-ness is important, could we maybe update this test to check that it is going last?